docs/docs/guides/developer_guides/getting_started.md

```diff
@@ -0,0 +1,252 @@
+
+```bash
+aztec start --sandbox
+```
+
+**Congratulations, you have just installed and run the Aztec Sandbox!**
+
+```bash
+     /\        | |
+    /  \    ___| |_ ___  ___
+   / /\ \  |_  / __/ _ \/ __|
+  / ____ \  / /| ||  __/ (__
+ /_/___ \_\/___|\__\___|\___|
+
+```
+
+In the terminal, you will see some logs:
+
+1. Sandbox version
+2. Contract addresses of rollup contracts
+3. PXE (private execution environment) setup logs
+4. Initial accounts that are shipped with the sandbox and can be used in tests
+
+You'll know the sandbox is ready to go when you see something like this:
+
+```bash
+[INFO] Aztec Server listening on port 8080
+```
+
+## Creating an account in the sandbox
+
+Now you have the sandbox running, let's create an account. Open a new terminal window and run:
+
+```bash
+aztec-wallet create-account
+```
+
+This will create a new account in your sandbox. You will see logs telling you the address, public key, secret key, and more.
+
+On successful depoyment of the account, you should see something like this:
+
+```bash
+New account:
+
+Address:         0x22d82b1ef045dfb4c3e0c4eac41fcf04ba29a3468d5ef5a123e4754064a4038d
+Public key:      0x24d07ed6baff3e12dd76e7299a771bfa430eed9e25b155e3253c958b0a93551f2012b9d0238581717e27b296124272053f53757942972df98ad6602369a4d2c92bdd202abb07ab331dd1e3d5595f00be5a1c9d3c3244e07453f38e83417b2a381a0039f104c8f2214611b7a645deb01bda7c79835a46cf6c78cd485bd4d1372a10844f6c07f18a8435c5998d7739be448c6b5c15688b7c228b0a7dc24b038de700d0d4af57aa6b07c4471bcdcc5228a17b1ac03f2aae84c33dd2933b6b5a1c70179399adacbfc836c8bf24741ea78d404fc803fe09cfc570560b16c1169a40be1f20f153e8e0e2f9856ee526723b49c6da81e82b382a8770e31c78f25a1784cc
+Secret key:     0x1a9af8f75b3430223b628f7b423667da14304d376a82cfd31dfe138d00a20c61
+Partial address: 0x198022f6ec9b51f3af4cdc731adf606c0143ad35f7947ad604db50e9310a31ac
+Salt:            0x0000000000000000000000000000000000000000000000000000000000000000
+Init hash:       0x04630e3997204b1c417868aa449beb272c3c3bf0efe22315c5920665a6390db7
+Deployer:        0x0000000000000000000000000000000000000000000000000000000000000000
+
+Waiting for account contract deployment...
+Deploy tx hash:  09245ec0d905a461c717d6f70600e3c13b8746ee97785fd491b58f5f56111854
+Deploy tx fee:   200013616
+```
+
+You can double check by running `aztec get-accounts`.
+
+You will be needing that `Address` and `Secret key` fields later! To make it easier to reference, export them like this:
+
+```bash
+export ACCOUNT_ADDRESS=<ADDRESS>
+export SECRET_KEY=<SECRET_KEY>
+```
+Make sure to replace the `<ADDRESS>` and `<SECRET_KEY>` with the actual results that were printed.
+
+With this new account, let's get some local test tokens!
+
+## Deploying a contract
+
+The sandbox comes with some contracts that you can deploy and play with. One of these is an example token contract. 
+
+Deploy it with this:
+
+```bash
+aztec deploy TokenContractArtifact --secret-key $SECRET_KEY --args $ACCOUNT_ADDRESS test-token TST 18
```

Comment:
Seems like we're not really using the CLI wallet here? It would be something like `aztec-wallet deploy --from accounts:my_wallet` etc (see the amazing alias there)